### ISSUE TITLE
Add message when there are no results

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/lefser#35]
using the `message()` function. This will help clarify the situation when there are no significant results. 
Also, add a unit test to check for this message, setting the threshold p-value small enough that there are no significant results.

**Comment by Peacesandy** [comment]
Okay, I'll start working on it

**Comment by lwaldron** [comment]
Here is a basis for the tests:

``` r
  suppressPackageStartupMessages(library(lefser))
  dataenv <- new.env(parent = emptyenv())
  data("zeller14", package = "lefser", envir = dataenv)
  zeller14 <- dataenv[["zeller14"]]
  zellersub <- zeller14[1:150, zeller14$study_condition != "adenoma"]
  zellersub <- relativeAb(zellersub)
  lefser(zellersub, groupCol = "study_condition", blockCol = "age_category", wilcox.threshold = 1e-10)
#> The outcome variable is specified as 'study_condition' and the reference category is 'control'.
#>  See `?factor` or `?relevel` to change the reference category.
#> Error in svd(X, nu = 0L): a dimension is zero
  lefser(zellersub, groupCol = "study_condition", blockCol = "age_category", kruskal.threshold = 1e-10)
#> The outcome variable is specified as 'study_condition' and the reference category is 'control'.
#>  See `?factor` or `?relevel` to change the reference category.
#> Error in filterKruskal(relab = relab_data, group = groupf, p.value = kruskal.threshold): There are no significant inter-group differences under the Kruskal-Wallis test at p < 1e-10
  lefser(zellersub, groupCol = "study_condition", blockCol = "age_category", lda.threshold = 1e6)
#> The outcome variable is specified as 'study_condition' and the reference category is 'control'.
#>  See `?factor` or `?relevel` to change the reference category.
#> [1] Names  scores
#> <0 rows> (or 0-length row.names)
```

<sup>Created on 2024-02-23 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by lwaldron** [comment]
Closed by 86139e8
